private/pkg/app/app.go

```diff
@@ -28,14 +28,18 @@ import (
 
 // EnvContainer provides environment variables.
 type EnvContainer interface {
-	// Env gets the environment variable value for the key.
+	// Env gets the environment variable raw string value for the key.
 	//
 	// Returns empty string if the key is not set or the value is empty.
 	Env(key string) string
 	// ForEachEnv iterates over all non-empty environment variables and calls the function.
 	//
 	// The value will never be empty.
 	ForEachEnv(func(string, string))
+	// GetEnvBoolValue gets and parses the environment variable bool value for the key.
+	//
+	// Returns false and the parse error if the key is not set or the value is empty.
+	GetEnvBoolValue(key string) (bool, error)
```

Comment:
I feel like this would be more useful if we passed a `defaultValue bool` argument here to indicate which value to use if the key isn't found in the environment.

Reply:
+1